src/transformers/models/maskformer/modeling_maskformer_swin.py

```diff
@@ -864,7 +864,9 @@ def __init__(self, config: MaskFormerSwinConfig):
         else:
             self.out_indices = tuple(i for i, layer in enumerate(self.stage_names) if layer in self.out_features)
         self.num_features = [config.embed_dim] + [int(config.embed_dim * 2**i) for i in range(len(config.depths))]
-        self.hidden_states_norms = nn.ModuleList([nn.LayerNorm(num_channels) for num_channels in self.channels])
+        self.hidden_states_norms = nn.ModuleList(
+            [nn.LayerNorm(num_channels) for num_channels in self.num_features[1:]]
```

Comment:
This is just a bug fix. `self.channels` isa list of the channel sizes for each of the output features. If `output_features` changes, then not all of the necessary `hidden_state_norms` layers are created